src/components/Trade/hooks/useTradeRoutes/useTradeRoutes.ts

```diff
@@ -18,13 +23,26 @@ export const useTradeRoutes = (): {
   const buyAsset = getValues('buyAsset')
   const sellAsset = getValues('sellAsset')
   const assets = useSelector(selectAssets)
-  const feeAsset = assets['eip155:1/slip44:60']
+  const feeAsset = assets[ETHEREUM_CAIP19]
 
   const setDefaultAssets = useCallback(async () => {
     try {
       const [sellAssetId, buyAssetId] = getDefaultPair()
       const sellAsset = assets[sellAssetId]
-      const buyAsset = assets[buyAssetId]
+      let isEthCaip19
+      let isEthereumChain
+
+      if (defaultBuyAssetId) {
+        isEthereumChain = assets[defaultBuyAssetId]?.chain === ChainTypes.Ethereum
+        isEthCaip19 = assets[defaultBuyAssetId]?.caip19 === ETHEREUM_CAIP19
+      }
+
+      // TODO: Currently, we only support ERC20 pairs but we might need to support cosmos pairs at some point.
```

Comment:
Nitpicks: 

1. Pairs can currently be ETH/ERC20 or ERC20/ERC20 
2. Cosmos pairs -> IBC pairs, since with Cosmos SDK, you can send/trade assets from all IBC chains. If using an exemple, I'd suggest mentioning e.g BTC Thorchain swaps though.

Other than this nitipick, I'd suggest clearly conveying intent in your comment: What's to-do? We might need to support more chains, so what's the work that actually need to get done to get this TODO removed?

Reply:
I updated the comment, it's a bit less precise on chains

Maybe I could do it already? Seems like it's not so over-engineered => I would do an array containing chains name that we support and just verify that the array contains the chain name